djangocms_versioning/datastructures.py

```diff
@@ -151,6 +164,12 @@ def _get_content_types(self):
 
     @cached_property
     def content_types(self):
+        """Get the primary key of the content type of the registered content model.
+
+        :return:  A list with one element - the pk of the content type of the content model
+        """
+        # TODO: Why on earth would this return a list which will always
+        # have just one element?
```

Comment:
This won't always have just one element - see PolymorphicVersionableItem._get_content_types.

Reply:
aaah, thanks! Will correct the docstring/comment.